week-4/A-objects-intro/exercise-part-2.js

```diff
@@ -6,16 +6,17 @@ The objects below have some syntax issues - try and fix them all!
 
 var kitten = {
     fur colour: "orange",

```

Comment:
You missed a syntax issue on this line. A reliable way to find them all is executing the file and resolving the parsing errors until there are none left.